addons/api/addon/handlers/indexed-db-handler.js

```diff
@@ -98,7 +98,7 @@ export default class IndexedDbHandler {
         );
 
         const dbRecords = paginateResults(indexedDbResults, page, pageSize).map(
-          this.indexedDb.normalizeData,
+          (item) => this.indexedDb.normalizeData(item, false),
```

Comment:
the item, along with an index number was being passed in instead of undefined. So it wasn't converting the `is_primary` field back to boolean type for each item (with indexes greater than 0)

Reply:
Ah good catch!